deployment/terraform/ssh/ssh.go

```diff
@@ -127,6 +127,33 @@ func (sshc *Client) UploadFile(src, dst string, sudo bool) ([]byte, error) {
 	return sshc.Upload(f, dst, sudo)
 }
 
+// Download downloads a given src remote filepath to a given dst writer.
+func (sshc *Client) Download(src string, dst io.Writer, sudo bool) error {
+	if strings.ContainsAny(src, `'\`) {
+		// TODO: copied from load-test repo. Need to be improved
+		// by using an actual sftp library.
+		return fmt.Errorf("shell quoting not actually implemented. don't use weird paths")
+	}
+
+	sess, err := sshc.client.NewSession()
+	if err != nil {
+		return err
+	}
+	defer sess.Close()
+	sess.Stdout = dst
+
+	cmd := fmt.Sprintf("cat '%s'", src)
```

Comment:
This cat business seriously needs to go away. Maybe our first help wanted ticket ?

Reply:
Yes. Ideally we should use scp/sftp but didn't feel like adding the dependency for a debug utility.